src/fides/api/schemas/privacy_notice.py

```diff
@@ -1,196 +1,27 @@
-from __future__ import annotations
+from typing import List, Optional
 
-from datetime import datetime
-from typing import Any, Dict, List, Optional
-
-from fideslang.models import Cookies as CookieSchema
 from fideslang.validation import FidesKey
-from pydantic import Extra, conlist, root_validator, validator
 
-from fides.api.custom_types import GPPMechanismConsentValue
 from fides.api.models.privacy_notice import ConsentMechanism, EnforcementLevel
-from fides.api.models.privacy_notice import PrivacyNotice as PrivacyNoticeModel
-from fides.api.models.privacy_notice import (
-    PrivacyNoticeFramework,
-    PrivacyNoticeRegion,
-    UserConsentPreference,
-)
 from fides.api.schemas.base_class import FidesSchema
 
 
-class GPPMechanismMapping(FidesSchema):
-    field: str
-    not_available: GPPMechanismConsentValue
-    opt_out: GPPMechanismConsentValue
-    not_opt_out: GPPMechanismConsentValue
-
-
-class GPPFieldMapping(FidesSchema):
-    region: PrivacyNoticeRegion
-    notice: Optional[List[str]]
-    mechanism: Optional[List[GPPMechanismMapping]]
-
-
-class PrivacyNotice(FidesSchema):
+class PrivacyNoticeHistorySchema(FidesSchema):
```

Comment:
All Privacy Notice schemas save this one were moved to Plus. Elements from this schema are used to build a consent email and are left here, including redefining some of the fields here.